package.json

```diff
@@ -103,6 +120,8 @@
     "rimraf": "^6.0.1",
     "rollup": "^4.22.5",
     "rollup-plugin-analyzer": "^4.0.0",
+    "rollup-plugin-dts": "^6.1.1",
+    "tslib": "^2.7.0",
```

Comment:
From my local testing, this looks like it is needed for array spread syntax, but I don't think we support any browsers or NodeJS versions that don't support array spread. 

```sh
[build:prod] (!) [plugin typescript] src/applications/apps.ts (103:11): @rollup/plugin-typescript TS2354: This syntax requires an imported helper but module 'tslib' cannot be found.
[build:prod] /Users/joeldenning/code/single-spa/src/applications/apps.ts:103:11
[build:prod] 
[build:prod] 103   return [...apps];
[build:prod]      
```

Setting the `target` field in `tsconfig.json` to es6 or later removes the need for tslib

```suggestion
```

Reply:
Set ES2015 as target for typescript.